### PR TITLE
Fix numpy deprecation warnings

### PR DESCRIPTION
**Pull request by adybbroe**
From Numpy 1.20 using the aliases of builtin types like np.int is deprecated. This PR fix that.
It also introduce the use of github actions instead of Travis-CI for CI-testing online.

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed 
 - [x] Passes ``flake8 pyorbital`` 
 - [ ] Fully documented

### CODE REVIEW

File: continuous_integration/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - behave
+  - mock
+  - zarr
+  - geoviews
+  - pytest
+  - pytest-cov
+  - fsspec
+  - pip
+  - pip:
+    - trollsift
```
Comment (djhoese) [inline continuous_integration/environment.yaml:30]: If I remember right the conda-incubator used in the actions for making the environment doesn't support pip installations right now. If these are passing in your fork, then nevermind.

Comment (djhoese) [reply continuous_integration/environment.yaml:30]: Oh maybe it was mamba that didn't support it.

File: setup.py [hunk 2]
```diff
@@ -31,8 +32,8 @@
       version=versioneer.get_version(),
       cmdclass=versioneer.get_cmdclass(),
       description='Orbital parameters and astronomical computations in Python',
-      author='Martin Raspaud, Esben S. Nielsen',
-      author_email='martin.raspaud@smhi.se',
+      author='The Pytroll Team',
+      author_email='pytroll@googlegroups.com',
```
Comment (djhoese) [inline setup.py:36]: Is this what we finally decided on? Or that slack channel thing @mraspaud talked about?

**Review by djhoese — COMMENTED** [review]
Looks good, but don't forget to update the badges on the README and maybe the RELEASING.md instructions if there are any.

File: continuous_integration/environment.yaml (hunk 1 above)
Comment (adybbroe) [reply continuous_integration/environment.yaml:30]: I think I have done this before in pyspectral and it seemed to work...

File: setup.py (hunk 2 above)
Comment (adybbroe) [reply setup.py:36]: No, it is just an incremental improvement

Comment (adybbroe) [reply setup.py:36]: We can wait till you/we have made your mind up in Satpy and then follow, ok?

Comment (djhoese) [reply setup.py:36]: Yeah, leaving it as you have it is fine by me.

**Comment by adybbroe** [comment]
Ok, I am ready with this now @djhoese . And the CI is green on my fork, thanks @mraspaud

**Review by djhoese — CHANGES_REQUESTED** [review]
Why not remove the entire travis file? And what about appveyor (assuming there is one)?

Update the README badges?

Also, because I noticed it on the python-geotiepoints project, what about adding a `long_description` to the setup.py that is the text of the README file?

**Comment by adybbroe** [comment]
> Looks good, but don't forget to update the badges on the README and maybe the RELEASING.md instructions if there are any.

Oh yeah, right! I remembered this morning, but forget it again! :-|

**Comment by adybbroe** [comment]
Ok @djhoese The setup fle is fixed. There is more to fix there, but that will be another PR then.
I removed the Travis file as suggested.

**Comment by adybbroe** [comment]
Concerning the badges @djhoese : There were none, now I added a few. Hope I got it right...

File: RELEASING.md [hunk 3]
```diff
@@ -6,18 +6,20 @@
...
-git tag v0.22.45 -m "Version 0.22.45"
+git tag -a v0.22.45 -m "Version 0.22.45"
 ```
 
 See [semver.org](http://semver.org/) on how to write a version number.
 
 6. push changes to github `git push --follow-tags`
-7. Verify travis tests passed and deployed sdist and wheel to PyPI
+
+7. Verify github actions passed and code deployed to PyPI
```
Comment (djhoese) [inline RELEASING.md:25]: You'll need to add something about making a github release here. git tag -> verify tests pass -> github release -> verify PyPI

File: setup.py [hunk 4]
```diff
@@ -42,10 +50,11 @@
                    "Topic :: Scientific/Engineering",
                    "Topic :: Scientific/Engineering :: Astronomy"],
       url="https://github.com/pytroll/pyorbital",
+      long_description=long_description,
```
Comment (djhoese) [inline setup.py:53]: Because the readme is markdown you'll also need:

```
long_description_content_type='text/markdown',
```

Comment (djhoese) [reply setup.py:53]: This makes it so PyPI includes the README text properly as a description of the package.

Comment (adybbroe) [reply setup.py:53]: Oh, @djhoese thanks for that! Didn't realize...

File: RELEASING.md (hunk 3 above)
Comment (adybbroe) [reply RELEASING.md]: Oh yes, thanks. Realize I haven't yet released after we switched from Travis to Github actions!

**Comment by adybbroe** [comment]
May I merge @djhoese and @mraspaud please?

**Pull request merged**